src/Components/Carousel/Carousel.js

```diff
@@ -18,7 +19,7 @@ const Carousel = () => {
 
   useEffect(async () => {
     const dataImage = await getImagesSlides();
-    setData(dataImage);
+    dataImage.length > 0 && setData(dataImage);
```

Comment:
please refactor this validation into a function, use as function name the response of this question:  what means dataImage.length > 0?
example:
```
isValidData = (data) => data.length >0
```

Reply:
Additionally data is a too generic name, use a more specific name, what this data mean? is an image? is an imageList?